asset/css/markbind.css

```diff
@@ -2,17 +2,24 @@ blockquote {
     font-size: 14px;
 }
 
-.radio-list-item label {
-    font-weight: inherit;
```

Comment:
Why was this removed?

Reply:
Apologies.. I was careless about removing the styling.